libr/core/cmd_log.c

```diff
@@ -180,15 +250,52 @@ static int cmd_log(void *data, const char *input) {
 	case '?':
 		r_core_cmd_help (core, help_msg_T);
 		break;
-	case 'T':
+	case 'T':  // Ts ? as ms?
 		if (r_config_get_i (core->config, "scr.interactive")) {
 			textlog_chat (core);
 		} else {
 			eprintf ("Only available when the screen is interactive\n");
 		}
 		break;
+	case '=':
+		if (input[1] == '&') {
+			if (input[2] == '&') {
+				r_cons_break_push (NULL, NULL);
+				while (!r_cons_is_breaked ()) {
+					r_core_cmd0 (core, "T=");
+					void *bed = r_cons_sleep_begin();
+					r_sys_usleep (100);
+					r_cons_sleep_end (bed);
+				}
+				r_cons_break_pop ();
+			} else {
+				// TODO: Sucks that we can't enqueue functions, only commands
+				eprintf ("Background thread syncing with http.sync started.\n");
+				RCoreTask *task = r_core_task_new (core, true, "T=&&", NULL, core);
+				r_core_task_enqueue (core, task);
+			}
+		} else {
+			if (atoi (input + 1) > 0 || (input[1] == '0')) {
```

Comment:
how is `T=` supposed to work? Does it expect a number as parameter? It's not clear from the help msg.

Reply:
yes, it expects a number but it's optional, in that case it pulls from the last id retrieved.